packages/scroller/test/scroller.test.js

```diff
@@ -139,4 +139,85 @@ describe('vaadin-scroller', () => {
       expect(scroller.getAttribute('overflow')).to.equal('top');
     });
   });
+
+  describe('scroll to', () => {
+    describe('vertical', () => {
+      beforeEach(() => {
+        scroller.scrollDirection = 'vertical';
+        scroller.style.maxHeight = '50px';
+        scroller.style.maxWidth = '50px';
+
+        const div = document.createElement('div');
+        div.textContent = 'Long text that does not fit Long text that does not fit Long text that does not fit';
+        scroller.appendChild(div);
+      });
+
+      it('should scroll to the bottom', () => {
+        scroller.scrollToEnd();
+        const scrollTopMax = scroller.scrollHeight - scroller.clientHeight;
+        expect(scroller.scrollTop).to.be.closeTo(scrollTopMax, 1);
+      });
+
+      it('should scroll to the top', () => {
+        scroller.scrollToEnd();
+        scroller.scrollToStart();
+        expect(scroller.scrollTop).to.equal(0);
+      });
+    });
+
+    describe('horizontal', () => {
+      beforeEach(() => {
+        scroller.scrollDirection = 'horizontal';
+        scroller.style.maxHeight = '50px';
+        scroller.style.maxWidth = '50px';
+
+        const div = document.createElement('div');
+        div.textContent = 'Long text that does not fit Long text that does not fit Long text that does not fit';
+        div.style.width = '200px';
+        scroller.appendChild(div);
+      });
+
+      it('should scroll to the right', () => {
+        scroller.scrollToEnd();
+        const scrollMax = scroller.scrollWidth - scroller.clientWidth;
+        expect(scroller.scrollLeft).to.equal(scrollMax);
+        expect(scroller.scrollLeft).to.not.equal(0);
+      });
+
+      it('should scroll to the left', () => {
+        expect(scroller.scrollLeft).to.equal(0);
+        scroller.scrollToStart();
+      });
+
+      it('should scroll to the left', () => {
+        expect(scroller.scrollLeft).to.equal(0);
+        scroller.scrollToStart();
+      });
+
+      describe('text direction RLT', () => {
+        beforeEach(() => {
+          scroller.setAttribute('dir', 'rtl');
+
+          // Scroll to center.
+          scroller.scrollLeft = -(scroller.scrollWidth - scroller.clientWidth) / 2;
+        });
+
+        it('should scroll to the left on scroll to end', () => {
+          scroller.scrollToEnd();
+          const scrollMax = scroller.scrollWidth - scroller.clientWidth;
+          expect(scroller.scrollLeft).to.equal(-scrollMax);
+          expect(scroller.scrollLeft).to.not.equal(0);
+        });
+
+        it('should scroll to the right on scroll to start', () => {
+          scroller.scrollToStart();
+          expect(scroller.scrollLeft).to.equal(0);
+        });
+
+        afterEach(() => {
+          scroller.removeAttribute('dir');
+        });
```

Comment:
Not necessary since `fixtureSync` automatically discards the instance after each test.